src/presentation/tabel-view.model.ts

```diff
@@ -4,9 +4,14 @@ import { bold } from "../deps.ts";
 
 export abstract class TableOutput implements TableConfig {
   public abstract info: string;
+
+  private isatty: boolean;
+
   constructor(
     readonly columns: string[],
-  ) {}
+  ) {
+    this.isatty = Deno.isatty(Deno.stdout.rid);
```

Comment:
yeah. I actually had the same idea. As this never changes and we could make some kind of global variable to do this. But I could not really find a good way to place it. Probably would need a standalone file to hold that value. It might actually be better to feed this value into the class via a ctor parameter so testing could be done more easily. Testing this either via global var or via this call kinda sucks. Long story short:

> Globally set var injected via ctor might be the way to go. 

If you find a good spot to place it feel free.